app/utils/prepareAnswer.js

```diff
@@ -18,6 +18,17 @@ function prepareStatsCompactAnswer(body, index, nameState) {
         data += '<i> (+' + numberWithIndianCommas(statedata['deltaconfirmed']) + ')</i>'
     }
     data += "\n\u{1F7E0} Active: " + numberWithIndianCommas(statedata['active'])
+    if (parseInt(statedata['deltarecovered']) > 0) {
+        let deltaActive = parseInt(statedata['deltaconfirmed']) - parseInt(statedata['deltarecovered']) - parseInt(statedata['deltadeaths'])
+        if (deltaActive > 0)
+            data += '<i> (+' + numberWithIndianCommas(deltaActive) + ')</i>'
+        else if (deltaActive < 0) {
```

Comment:
This else statement is not even needed because, in the API, they aren't updating negative changes except for `Unknown State`. We can drop this else part and we're good to go. Great work, thanks for the PR.

Reply:
delta active is not actually there in the API. I calculated the deltaActive using 
deltaconfirmed-deltarecovered-deltadeaths
else part was needed to add the minus(-) sign while displaying change in number of active cases as it is not necessary that active cases will always rise. for example take a look at this example where minus sign was needed to show decrease in active cases number which is not possible using 'if' part as it was applying commas to minus sign as well and was producing output as: +-,104
![odisha](https://user-images.githubusercontent.com/57032785/84057772-c6ffc300-a9d5-11ea-8a5d-6e6b0e76de4b.png)

Reply:
Yeah, good idea. Can you create a new PR for this? Just for keeping the code history organised.

Reply:
I made one, please guide if you find anything wrong. Thanks

Reply:
I mean remove this change from this PR and create a new PR with that new change (do it after this one is merged), I will create an issue for this also.

Reply:
#31

Reply:
oh okay i got it, thanks i will do it now

Reply:
ok i removed the deltaActive part, have a look and merge if ok💯